tests/test_utils.py

```diff
@@ -174,3 +179,44 @@ def test_collect_args_local(wps_test_collect_args, file1, file2, argc):
 )
 def test_collect_args_online(wps_test_collect_args, file1, file2, argc):
     collect_args_test(wps_test_collect_args, file1, file2, argc)
+
+
+@pytest.mark.parametrize(
+    ("package"), [("base"), ("utils")],
+)
+def test_get_package(package):
+    pkg = get_package(package)
+    assert pkg.__dict__["__rname__"] == package
```

Comment:
A clever way to check if an R package class is returned. :+1:

Reply:
I just copied what you had :smile: